lib/server.js

```diff
@@ -270,6 +271,7 @@ function create(log, error, config, routes, db, translator) {
       const acceptLanguage = trimLocale(request.headers['accept-language'])
       request.app.acceptLanguage = acceptLanguage
       request.app.locale = translator.getLocale(acceptLanguage)
+      userAgent.call(request.app, request.headers['user-agent'])
```

Comment:
FWIW, I like this "decorate the request" pattern, and feel like we should do it for more stuff, such as the ip geolocation.  It would be neat to be able to do it in a lazy way using a computed property of some kind, but NBD if we do this bit of parsing every time.